metadata.rb

```diff
@@ -1,12 +1,12 @@
 name 'chef-compliance'
-maintainer 'Brian Menges'
-maintainer_email 'mengesb@users.noreply.github.com'
+maintainer 'Sous Chefs'
+maintainer_email 'help@sous-chefs.io'
```

Comment:
I don't know what will be the final domain. But currently the correct and working domain is [sous-chefs.org](http://sous-chefs.org/).